build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.1'
+//        classpath 'com.android.tools.build:gradle:2.2.2'
```

Comment:
Sorry JJ, Android Studio kept giving me errors every time I deleted the line so I was unsure of whether I needed to completely remove the line.